client/packages/system/src/Encounter/DetailView/Footer.tsx

```diff
@@ -120,12 +123,23 @@ export const Footer: FC<FooterProps> = ({
               startIcon={<SaveIcon />}
               label={t('button.save')}
             />
-            {encounter?.status && (
-              <StatusChangeButton
-                currentStatus={encounter.status}
-                onSave={onChangeStatus}
-              />
-            )}
+            {encounter?.status &&
+              // Status no longer editable once visited
+              // Instead show schedule next encounter button
+              (encounter.status === EncounterNodeStatus.Visited ? (
```

Comment:
First side quest, per PR description - I think best for the user to be able to get back to the schedule next encounter window when they are ready 👀

Reply:
Does this also appear when next encounter is already scheduled ?

Reply:
It does... but you can check if there are existing future encounters in the `More` panel. The modal that pops up after `Mark as Visited` also doesn't check for existing scheduled... agree it's probably worth doing something about - can it be future issue?

Reply:
#7471